crates/matrix-sdk-crypto/src/olm/account.rs

```diff
@@ -511,6 +510,13 @@ pub struct StaticAccountData {
     creation_local_time: MilliSecondsSinceUnixEpoch,
 }
 
+#[cfg(not(tarpaulin_include))]
+impl std::fmt::Debug for StaticAccountData {
```

Comment:
I think we can even derive now, there's nothing secret there, perhaps the `creation_local_time` isn't that interesting.